spec/unit/util/common_events/processor_spec.rb

```diff
@@ -33,7 +40,7 @@
         end
 
         it 'returns correct object types' do
-          described_class.find_each(procs_dir).each do |processor|
+          described_class.find_each(procs_dir) do |processor|
```

Comment:
Possible conflict with the subject processor?

Reply:
Good catch. Name changed to avoid conflict.